nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ValidateDcom.java

```diff
@@ -322,11 +324,10 @@ private void dumpStack(Throwable t) {
 
     private boolean testPort(int port, String description) {
         try {
-            Socket socket = new Socket();
-            socket.connect(new InetSocketAddress(host, port), 4000);
-            socket.close();
-            out.append(Strings.get("validate.dcom.connect",
-                    description, port, host)).append('\n');
+            try (Socket socket = new Socket()) {
```

Comment:
If you do this at the outer try the IOException will also be caught with the outer catch. Inner try-with-resources is obsolete.